### PR TITLE
Homepage added

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Homepage

Live Preview: https://deploy-preview-3--zeplins.netlify.app/views/home

### CODE REVIEW

File: public/css/home.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+:root{
+    --food-slider-bg:#171A29;
+    --food-top-brands:#F8F8F8;
+    --card-bg:#F5F5F5;
+    --border-black-op: rgba(0,0,0,0.1);
+}
+.nav-zindex{
+    z-index: 999;
+}
```
Comment (j24m) [inline public/css/home.css:11]: no space is there over here.

File: public/css/home.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+.container{
+    width: 95%;
+    margin: auto;
+}
+
```
Comment (j24m) [inline public/css/home.css:16]: this is fine .

File: public/css/home.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+.navbar-bg{
+  padding: 1rem 1rem 0;
+  border-bottom: 1px solid var(--border-black-op);
+
+}
+
```
Comment (j24m) [inline public/css/home.css:21]: fix : but then why 2 spaces here ? either leave 1 or 2 spaces , and follow that everywhere.

**Review by Samarth-5109 — APPROVED** [review]
Looks good to merge

**Review by j24m — APPROVED** [review]
Code seems fine , good work !!

**Comment by mehulsatardekar** [comment]
@j24m Did all the changes thank you for pointing out the indentation issue

**Pull request merged**